RM_vs_DFN.ipynb

```diff
@@ -0,0 +1,337 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.        # plt.ylabel("Discharge capacity [A.h]")
</pre><p>If these lines are not needed, let's delete them.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/brosaplanella/formation/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
Deleted them